extensions/HUAWEI/SPV_HUAWEI_cluster_culling_shader.adoc

```diff
@@ -12,23 +12,23 @@ https://github.com/KhronosGroup/SPIRV-Registry
 
 == Contributors
 - Cheng Wei, HUAWEI
+- Jinyongjie, HUAWEI
 - Yuchang Wang, HUAWEI
 
 == Status
-Draft
+Complete.
 
 == Version
 [width="40%",cols="25,25"]
 |========================================
-| Last Modified Date | 2022-11-16
-| Revision           | 1
+| Last Modified Date | {docdate}
 |========================================
 
 Dependencies
 This extension is written against the SPIR-V Specification, Version 1.6
 
 == Overview
-This extensions provides SPIR-V support for the *GLSL GL_HUAWEI_cluster_culling_shader* extension which adds a new programmable shader types - cluster culling shaders - that has an execution environment similar to that of compute shaders. The developer can use this new shader to perform coarse-level geometry culling on GPU and direct emit visible cluster to the subsequent rendering pipeline.
+This extensions provides SPIR-V support for sthe *GLSL GL_HUAWEI_cluster_culling_shader* extension which adds a new programmable shader types - cluster culling shaders - that has an execution environment similar to that of compute shaders. The developer can use this new shader to perform coarse-level geometry culling on GPU and direct emit visible cluster with a shading rate to the subsequent rendering pipeline.
```

Comment:
Minor typo:
```suggestion
This extensions provides SPIR-V support for the *GLSL GL_HUAWEI_cluster_culling_shader* extension which adds a new programmable shader types - cluster culling shaders - that has an execution environment similar to that of compute shaders. The developer can use this new shader to perform coarse-level geometry culling on GPU and direct emit visible cluster with a shading rate to the subsequent rendering pipeline.
```

Reply:
thank you Ben, I fixed this typo, please help to merge this change.